Yosemite/Yosemite/Stores/ProductReviewStore.swift

```diff
@@ -81,6 +81,10 @@ private extension ProductReviewStore {
                 return
             }
 
+            if pageNumber == 1 {
```

Comment:
nit: we usually compare with `Default.firstPageNumber` like in `ProductCategoryStore`